imapclient/fetch.go

```diff
@@ -1030,7 +1020,11 @@ func readBodyFldParam(dec *imapwire.Decoder) (map[string]string, error) {
 			if params == nil {
 				params = make(map[string]string)
 			}
-			params[k] = s
+			decoded, err := options.DecodeText(s)
```

Comment:
CI is unhappy, seems like there is a typo here, should be `decodeText`.

Reply:
Oops yes made a typo, sorry